### ISSUE TITLE
Create package does not have the include source checkbox

### ISSUE THREAD

**Issue by fiolavzw** [dnnsoftware/Dnn.AdminExperience#1061]
## Description of bug
DNN 9.3 does not have the checkbox "include source"

## Steps to reproduce
List the steps to reproduce the behavior:
1. Go to 'host-extentions'
2. Click on 'edit an extention'
3. click on the create package button top-right
4. bug: filelist is incorrect and always has the .vb files in it and there is no checkbox te exclude the source files

## Current result
file list with all files
check box NOT available on screen "include source"

## Expected result
file list without the .vb
check box available on screen "include source"

## Screenshots
![image](https://user-images.githubusercontent.com/49470612/58860850-57ae0f80-86ad-11e9-9b70-b57178c3c20e.png)

## Error log
none

## Additional context
none

## Affected version

* [x] 9.3.2
* [x] 9.3.1
* [] 9.2.2
* [] 9.2.1
* [] 9.2
* [] 9.1.1
* [ ] 9.1
* [ ] 9.0

## Affected browser

* [x ] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Internet Explorer
* [ ] Edge

**Comment by valadas** [comment]
Correct me if I am wrong, but I think that behaviour changed to just include everything in the module folder as if "include source" was checked but the message never got changed...

**Comment by sleupold** [comment]
IMO it makes sense to create an install package without source, i.e. the "include source" is missing.

**Comment by fiolavzw** [comment]
This functionality is used a lot by developers who want to publish their custom made modules. Source code should never be installed/uploaded to a sever where the websites actually run, so the default behavior should be that it does not include the source code. (in my opinion)

**Comment by valadas** [comment]
I agree with this. One thing to keep in mind though is that this was built in a time where we only had to deal with webforms. With modern modules now, there could be js files that are runtime and js files that are just for build time (just one example). So how would we detect what is a source file or not a source file in this scenario?

**Comment by sleupold** [comment]
there are two options: conventions or configuration.
Convention could be excluding some folders, 
Configuration would be a kind of .ignore file

**Comment by valadas** [comment]
Yep, both options would need developers to follow a certain pattern though...
